### DISCUSSION TITLE
Can someone guide where the .env file?

### DISCUSSION THREAD

**Discussion by kronicklez** [akvorado/akvorado#1397] [Q&A]
Hi expert,

I just deploy akvorado using docker method. I'm read https://demo.akvorado.net/docs/intro#introduction. I'm not understand "Remove the reference for docker-compose-demo.yml from .env," . Where is located .env file?

Once running, Akvorado web interface should be running on port 8081. A few synthetic flows are generated in the background. To disable them:

1.) Remove the reference for docker-compose-demo.yml from .env,
2.) Comment the last line of akvorado.yaml, and
3.) Run docker compose up -d --remove-orphans.

Appreciate someone guide.

**Comment by jake1683** [comment]
Its located in the main installation directory of Akvorado.  Its a hidden file.

**Comment by kronicklez** [comment]
Hi Jake / Expert member,

Thanks for your guide. 

root@netflow-akvorado:~/akvorado# ls -a
.  ..  config  docker  .env  .env.dist  orchestrator

After all the 3 step above then i just need to change some info in file "akvorado.yaml" and "inlet.yaml" right? Thanks appreciate confirmation.

root@netflow-akvorado:~/akvorado/config# ls
akvorado.yaml  console.yaml  demo.yaml  inlet.yaml
